src/main/webapp/app/exercises/programming/hestia/git-diff-report/git-diff-report-modal.component.ts

```
@@ -2,150 +1,152 @@
+        } catch (e) {
+            this.errorWhileFetchingRepos.set(true);
+        }
     }
 
     close(): void {
         this.activeModal.dismiss();
     }
 
     private calculateTemplateMapKey() {
-        return this.report.programmingExercise.id! + '-template';
+        return this.report().programmingExercise.id! + '-template';
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)